.github/workflows/python-tests.yml

```diff
@@ -83,7 +104,7 @@ jobs:
             path: python/dist/
```

Comment:
(I can't leave a comment on lines 94 and 97, or I'd say this there) Also, should we update the 3.9 to use the python version matrix?

Reply:
I decided that hardcoding the value was the lesser evil when compared to duplicating the awk line, which will match the 3.9 value. If we expected to update that value, then I'd be inclined to duplicate the awk command.